processing/src/main/java/org/apache/carbondata/processing/util/CarbonLoaderUtil.java

```diff
@@ -363,7 +368,7 @@ public static String getEscapeChar(String escapeCharacter) {
 
 
   public static void readAndUpdateLoadProgressInTableMeta(CarbonLoadModel model,
-      boolean insertOverwrite) throws IOException {
+      boolean insertOverwrite, String uuid) throws IOException {
```

Comment:
Better don't change the method, add a new method with UUID and call the same method internally.